.gitignore

```diff
@@ -60,6 +63,18 @@
 # build
 *build
```

Comment:
This line should probably be changed to:
```.gitignore
/build/
```
which will only match a directory named `build` in the same folder as this `.gitignore` file

Reply:
yeah, this is because I was using a `cbuild` directory locally to test the manual cmake build